engine.py

```diff
@@ -506,7 +506,14 @@ def start_interactive(self):
                 self.material_dict[mat_slot.material].append(obj)
         
         # export rib and bake
-        write_rib(self, self.scene, self.ri)
+        
+        
+        #Check if rendering select objects only.
+        if(self.scene.renderman.render_Selected_Objects_Only):
```

Comment:
Minor point but pep8 would say to use render_selected_objects_only. Note capitlization